drivers/wifi/esp/esp_offload.c

```diff
@@ -258,7 +258,7 @@ static int _sock_send(struct esp_data *dev, struct esp_socket *sock)
 	k_sem_give(&dev->iface_data.rx_sem);
 
 	/* Wait for '>' */
-	ret = k_sem_take(&dev->sem_tx_ready, 5000);
+	ret = k_sem_take(&dev->sem_tx_ready, K_MSEC(5000));
```

Comment:
`K_SECONDS(5)` would be more appropriate (I realize it's hard/impossible for the script to catch these)